docs/mastery.md

```diff
@@ -0,0 +1,30 @@
+# CC Final Project - Mastery of Cloud Persona and Tools
+
+## Chosen Persona
+
+For this project, we decided to take on the persona of the **Dev-Ops Engineer**, while also dabbling
+a bit in the **AI Engineer** persona. Dev-Ops engineers are in charge of creating solutions that
+assist in the development process, and that is what we aimed to accomplish. Having good communication
+between team members is key when developing any software application. This communication is hindered
+when team members with disabilities like _prosopagnosia_ are unable to recognize the faces of their
+peers, which can lead to confusion for all parties involved when discussing the team's work. Because
+video chats are so common these days due to the ongoing pandemic, we decided to create a solution
+that would alleviate these issues in a video chat environment. Thus, we created a live facial
+recognition tool with the potential to integrate into video chats to better streamline the
+communication between team members and make the team environment more accessible.
+
+Naturally, given that this is a facial recognition tool, there is also some machine learning
+involved, hence the **AI Engineer** persona that was touched on earlier. However, given that we
+utilized a library that took care of most of the machine learning for us, the **AI Engineer**
+persona was not the primary focus of this project.
+
+## Tools and Technologies
+
+To keep true to the assignment at hand, we deployed our site through Azure Websites to deliver a
+responsive experience to the user. Also, in order to boost the performance of the facial recognition,
+the machine learning models for facial detection were loaded in the cloud prior to deployment, thus
+making use of the distributed computing and speed of Azure. The intent was to keep to the concept of
+_facilitating communication and collaboration_, which is one of the key concepts highlighted by the
+Azure Dev-Ops certification page. Leveraging cloud resources to create a clean, fast, and responsive
+user experience, we did just that.
```

Comment:
Looks good! I can probably add more details here about the specifics if you want. Or we can save that for the presentation I guess, probably not a huge deal